includes/Shopping/Woocommerce_Query.php

```diff
@@ -115,6 +128,11 @@ public function get_search( string $search_term ) {
 	protected function get_product_images( int $image_id ): array {
```

Comment:
Aside: why is this called `get_product_images` when this just returns a single url+alt tuple? It does not return multiple _images_.

We should rename this.

Reply:
Updated